brave-core/src/main/java/com/github/kristofa/brave/SpanCollectorMetricsHandler.java

```diff
@@ -1,14 +1,14 @@
-package com.github.kristofa.brave.scribe;
+package com.github.kristofa.brave;
 
 /**
- * Monitor {@link ScribeSpanCollector} by implementing reactions to these events, e.g. updating suitable metrics.
+ * Monitor SpanCollector by implementing reactions to these events, e.g. updating suitable metrics.
```

Comment:
supernit `{@linkplain SpanCollector}` to make this refactor-safe, or change to "span collector" to disconnect from type names.